packages/tocco-ui/src/EditableValue/typeEditorFactory.js

```diff
@@ -17,6 +17,7 @@ import MultiRemoteSelect from './typeEditors/MultiRemoteSelect'
 import SearchFilterEdit from './typeEditors/SearchFilterEdit'
 import UrlEdit from './typeEditors/UrlEdit'
 import PhoneEdit from './typeEditors/PhoneEdit'
+import Duration from './typeEditors/Duration'
```

Comment:
Name component and file DurationEdit to match naming convention. Names should be predictable and self-explaining.

Reply:
Good point